kairon/shared/data/data_models.py

```
@@ -0,0 +1,1341 @@
+                and v[i].type != StoryStepType.form_end
+            ):
+                raise ValueError(
+                    f"Only {StoryStepType.form_end} step type can have empty name"
+                )
+            if v[i].type == StoryStepType.stop_flow_action and i != len(v) - 1:
+                raise ValueError("Stop Flow Action should only be at the end of the flow")
+            if v[i].type == StoryStepType.intent and v[j].type == StoryStepType.stop_flow_action:
+                raise ValueError("Stop Flow Action should not be after intent")
+
+        if "type" in values:
+            if values["type"] == StoryType.rule and intents > 1:
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)